demo/lib/host.bash

```diff
@@ -3,6 +3,7 @@ source "$(dirname "${BASH_SOURCE[0]}")/command.bash"
 HOST_PROMPT=${HOST_PROMPT-"\e[38;5;11mhost>\e[0m "}
 HOST_LIB_DIR="$(dirname "${BASH_SOURCE[0]}")"
 HOST_PROJECT_DIR="$(dirname "$(dirname "$(realpath "$HOST_LIB_DIR")")")"
+HOST_GORESCTRL_DIR="$(realpath "$HOST_PROJECT_DIR/../goresctrl")"
```

Comment:
nitpick: should this be allowed to be overridden ?

Reply:
Thanks! Overriding is possible now.